In-house patch to remove unnecessary dependencies from Keystone's
requirements files. The specific reasons are as follows:

paste			Not applicable

posix_ipc		Not applicable

--- keystone-2015.1.2/keystone.egg-info/requires.txt.orig      2015-10-13 13:44:44.730755549 -0600
+++ keystone-2015.1.2/keystone.egg-info/requires.txt        2015-10-13 13:44:50.161960656 -0600
@@ -4,7 +4,6 @@ eventlet!=0.17.0,>=0.16.1
 greenlet>=0.3.2
 netaddr>=0.7.12
 PasteDeploy>=1.5.0
-Paste
 Routes!=2.0,>=1.12.3
 cryptography>=0.8 # Apache-2.0
 six>=1.9.0
@@ -29,5 +28,4 @@ pysaml2
 dogpile.cache>=0.5.3
 jsonschema<3.0.0,>=2.0.0
 pycadf<0.9.0,>=0.8.0
-posix-ipc
 msgpack-python>=0.4.0

--- keystone-2015.1.2/requirements.txt.orig  2015-10-13 13:43:45.598126189 -0600
+++ keystone-2015.1.2/requirements.txt    2015-10-13 13:43:57.481207228 -0600
@@ -8,7 +8,6 @@ eventlet!=0.17.0,>=0.16.1
 greenlet>=0.3.2
 netaddr>=0.7.12
 PasteDeploy>=1.5.0
-Paste
 Routes!=2.0,>=1.12.3
 cryptography>=0.8 # Apache-2.0
 six>=1.9.0
@@ -33,5 +32,4 @@ pysaml2
 dogpile.cache>=0.5.3
 jsonschema<3.0.0,>=2.0.0
 pycadf<0.9.0,>=0.8.0
-posix-ipc
 msgpack-python>=0.4.0
